frontend/src/lib/components/SelectGradientOverflow.tsx

```diff
@@ -46,6 +46,7 @@ export type SelectGradientOverflowProps = SelectProps<any> & {
     delayBeforeAutoOpen?: number
     dropdownMatchSelectWidth?: boolean | number
     placement?: 'bottomLeft' | 'topLeft' // Dropdown placement (undefined = auto). See API at https://ant.design/components/dropdown
+    handleBlur?: () => void
```

Comment:
I get a curious React error which tells me that handleBlur should be lowercase

![image](https://user-images.githubusercontent.com/85295485/127502621-d5d19b21-a906-4a54-a87f-da02ca513874.png)

But I'm not sure if I've done something more fundamentally wrong - so didn't want to just change it to be lowercase

Reply:
Oh I see what the problem is, `handleBlur` was being passed down all the way to the raw HTML component through `props` because it wasn't destructured. See the commit I pushed, we extract the `handleBlur` property so it doesn't get rendered in the downstream components.

Reply:
Ah nice catch.